howso/visuals/visuals.py

```diff
@@ -581,3 +581,48 @@ def plot_interpretable_prediction(
 
     if len(figures) == 1:
         return figures[0]
+
+
+def plot_fairness_disparity(
+    fairness_results: dict,
+    ref: str,
+    threshold: float = 0.75,
+    x_rotate: bool = False
+):
+    """
+    Helper function for plotting fairness disparity results.
+
+    Parameters
+    ----------
+    fairness_results : dict
+        Dictionary of the fairness disparity ratios.
+    ref : str
+        The reference class for the metric calculation.
+    threshold : float, default 0.75
+        Threshold for values to be classified as fair. Values below this threshold are colored red when
+        graphing while values above are colored green.
+    x_rotate : bool, default False
+        Whether to rotate the x-axis labels.
+    """
+    num_subplots = len(fairness_results)
+    fig = make_subplots(rows=1, cols=num_subplots, subplot_titles=list(fairness_results.keys()))
+
+    for i, (_, results) in enumerate(fairness_results.items(), 1):
+        sorted_data = {key: results[key] for key in sorted(results, key=lambda x: (x != ref, x))}
+        for key, value in sorted_data.items():
+            color = 'grey' if key == ref else ('red' if value < threshold else 'green')
```

Comment:
Add two keyword-only arguments to the function: `fair_color` and `unfair_color`, with defaults set to these colors.

That being said, I think the colors may be better off being something like

`#B3589A` for the "red" (it's actually a purple) and `#9BBf85` for the green. This might help out folks with red/green colorblindness.

Reply:
Just remembered there's a grey as well, I'm not 100% sure what a good name for that would be but it's probably worth being a keyword-only argument as well.